PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,64 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean isfirstRun;
+
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
+        isfirstRun = settings.getBoolean("firstRun", true);
+        if (isfirstRun) {
+            addSlide(new IntroFragment(R.layout.intro_screen_1));
+            addSlide(new IntroFragment(R.layout.intro_screen_2));
+            addSlide(new IntroFragment(R.layout.intro_screen_3));
+            addSlide(new IntroFragment(R.layout.intro_screen_4));
+            addSlide(new IntroFragment(R.layout.intro_screen_5));
+            addSlide(new IntroFragment(R.layout.intro_screen_6));
+        } else {
+            loadMainActivity();
+        }
+    }
+
+    private void loadMainActivity() {
+        settings = getSharedPreferences("prefs", 0);
+        SharedPreferences.Editor sharedPreferencesEditor = settings.edit();
+        sharedPreferencesEditor.putBoolean("firstRun", false);
+        sharedPreferencesEditor.commit();
+        Intent startPage = new Intent(this, StartActivity.class);
```

Comment:
Can you call this `startPageIntent`? So it's clear what it is (sorry for making you change it multiple times :P )